src/github_review_prompts/utils/resolution_master.py

```
@@ -0,0 +1,606 @@
+"""範囲外コメント解決システムのマスターコントローラー"""
+
+import logging
+from typing import Dict, List, Optional, Any, Tuple
```

Ruff (0.12.2)

`typing.Tuple` imported but unused

Remove unused import: `typing.Tuple`

(F401)